seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -444,71 +628,76 @@ def test_create_project_from_workspace(self, mock_tempfile, mock_mv_file, mock_s
         self.assertEqual(responses.calls[3].request.headers['Authorization'], 'Bearer {}'.format(MOCK_TOKEN))
         self.assertEqual(responses.calls[3].response.json(), UPDATE_DAG_TASKS_RESP)
 
+        call_cnt = 4
```

Comment:
this would be cleaner if you didn't abbreviate t and also set it explicitly here instead of in the if block: 
```
call_count = 5 if test_add_data else 4
```